### DISCUSSION TITLE
Lightning Terminal App Help Please

### DISCUSSION THREAD

**Discussion by BonzoDogDooDah** [lightninglabs/lightning-terminal#664] [General]
Hi All,

I used the Lightning Termminal App the other day to do a loop out but I did not see where to choose which channel to pick to perform the loop out as that tab never presented itself? So I did a loop out anyway and it sent 250k sats to my BTC wallet after more than 1 hour. I can’t see where the 250k sats came from as there was no increase in inbound liquidity to any of our channels or a decrease in any? I realize we are advanced newbies but if anyone knows what happened and how we can possibly do a loop out properly next time we would really appreciate it. 

The Lightning Terminal App in Umbrel does not seem to have all the features that the CLI version does.

Sorry but another thing, is it possible to use the Lightning Terminal’s Auto Loop feature like the CLI version has when using the Lightning Terminal App in Umbrel? I did not see where this could be done?

Thank you

Orange_Bonzo

**Comment by TommyBuho** [comment]
Not much help but I am also interested in Umbrel + Lightning Terminal details. It is not intuitive so far. 

TB
